src/main/sensors/battery.c

```diff
@@ -67,7 +67,7 @@
  *
  */
 
-#define VBAT_STABLE_MAX_DELTA 20
+#define VBAT_STABLE_MAX_DELTA 50 //50mV
```

Comment:
A value of 10 works better in detecting the peak.
10 means a difference of less than 100mv.